packages/components/src/toolbar/toolbar-button/toolbar-button-container.tsx

```diff
@@ -0,0 +1,14 @@
+/**
+ * Internal dependencies
+ */
+import type { ToolbarButtonContainerProps } from './types';
+import type { WordPressComponentProps } from '../../ui/context';
+
+const ToolbarButtonContainer = ( {
+	className,
+	children,
+}: WordPressComponentProps< ToolbarButtonContainerProps, 'div', false > ) => (
```

Comment:
This component doesn't pass down the rest props to the `div`, so we'll need to explicitly add just `className` and `children` to `ToolbarButtonContainerProps`.

```suggestion
const ToolbarButtonContainer = ( {
	className,
	children,
}: ToolbarButtonContainerProps ) => (
```

Reply:
Done with [e7ae519](https://github.com/WordPress/gutenberg/pull/47750/commits/e7ae519baf9e6429fc15989588a6b5c612ca5eb3) and [d29a6bf](https://github.com/WordPress/gutenberg/pull/47750/commits/d29a6bf9973b147f2de214804fadb416033b478e).